app/assets/stylesheets/tables-report.css.scss

```diff
@@ -7,6 +7,11 @@
 @forward 'usa-prose';
 @forward 'usa-table';
 @forward 'usa-alert';
+@forward "uswds-utilities";
```

Comment:
This one line of code will more than double the size of the stylesheet (60kb to 140kb), and email styles are embedded with every message sent. See previous #7224.

If you need additional utility classes, I'd suggest doing what's done in `email.css.scss` to copy in individual utilities.

Example:

https://github.com/18F/identity-idp/blob/8bd41a8c835b57ab32cd79aaf5063fa9a2606b1b/app/assets/stylesheets/email.css.scss#L215-L217

Reply:
Updates made to include as needed instead of bringing in the whole schebang.